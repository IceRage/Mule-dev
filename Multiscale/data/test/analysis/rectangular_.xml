<?xml version="1.0" encoding="utf-8"?>
<!--Warning! This xml file was automatically generated by a C++ program using the Boost PropertyTree library.-->
<experiment>
	<timepoint>
		<spatialEntity>
			<pseudo3D type="region">
				<clusteredness>0.9980355778675106</clusteredness>
				<density>0.3090113050567821</density>
				<area>27129.5</area>
				<perimeter>757.760892868042</perimeter>
				<distanceFromOrigin>316.227766016838</distanceFromOrigin>
				<angle>36.47418500219597</angle>
				<shape>triangular</shape>
				<triangleMeasure>0.8064085844117723</triangleMeasure>
				<rectangleMeasure>0.5934254174991814</rectangleMeasure>
				<circleMeasure>0.3237010885035476</circleMeasure>
				<centroid>
					<x>1141.03</x>
					<y>900.4112</y>
				</centroid>
			</pseudo3D>
		</spatialEntity>
		<spatialEntity>
			<pseudo3D type="region">
				<clusteredness>0.9901557039492034</clusteredness>
				<density>0.4901932634774364</density>
				<area>240922</area>
				<perimeter>2153.12531709671</perimeter>
				<distanceFromOrigin>108.6324076875773</distanceFromOrigin>
				<angle>109.7778345325739</angle>
				<shape>rectangular</shape>
				<triangleMeasure>0.6334815468757613</triangleMeasure>
				<rectangleMeasure>0.706133489065172</rectangleMeasure>
				<circleMeasure>0.5624248149434126</circleMeasure>
				<centroid>
					<x>314.9832</x>
					<y>698.8802</y>
				</centroid>
			</pseudo3D>
		</spatialEntity>
		<numericStateVariable>
			<name>avgClusterednessRegions</name>
			<value>0.001176089756054817</value>
		</numericStateVariable>
		<numericStateVariable>
			<name>avgDensityRegions</name>
			<value>0.3996022842671093</value>
		</numericStateVariable>
	</timepoint>
</experiment>
