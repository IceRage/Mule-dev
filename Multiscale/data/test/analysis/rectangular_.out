Clusteredness degree,Density,Area,Perimeter,Distance from origin,Angle(degrees),Shape,Triangle measure,Rectangle measure,Circle measure,Centre (x-coord),Centre (y-coord)
0.998036,0.309011,27129.5,757.761,316.228,36.4742,triangular,0.806409,0.593425,0.323701,1141.03,900.411
0.990156,0.490193,240922,2153.13,108.632,109.778,rectangular,0.633482,0.706133,0.562425,314.983,698.88

Average clusteredness degree: 0.00117609
Average density: 0.399602
